### PR TITLE
chore(sdk-coin-bsc): rename bsc to bnb

### PR DESCRIPTION
**Pull request by kxl4126**
Ticket: EA-709

This is a breaking change for services that rely on SDK BSC constants like WP. Will verify corresponding WP changes with alpha release.

### CODE REVIEW

File: modules/account-lib/src/index.ts [hunk 1]
```diff
@@ -120,8 +120,8 @@ const coinBuilderMap = {
   tcelo: Celo.TransactionBuilder,
   avaxc: AvaxC.TransactionBuilder,
   tavaxc: AvaxC.TransactionBuilder,
-  bsc: Bsc.TransactionBuilder,
-  tbsc: Bsc.TransactionBuilder,
+  bsc: Bnb.TransactionBuilder,
+  tbsc: Bnb.TransactionBuilder,
```
Comment (mmcshinsky-bitgo) [inline modules/account-lib/src/index.ts:124]:
```suggestion
  bnb: Bnb.TransactionBuilder,
  tbnb: Bnb.TransactionBuilder,
```

**Review by mmcshinsky-bitgo — CHANGES_REQUESTED** [review]
All these commits will need to be marked as breaking changes in the commit footer. Commit descriptions should also outline the breaking change for external consumers.

File: modules/sdk-coin-bsc/README.md [hunk 2]
```diff
@@ -16,11 +16,11 @@ Next, you will be able to initialize an instance of "bitgo" through `@bitgo/sdk-
 
 ```javascript
 import { BitGoAPI } from '@bitgo/sdk-api';
-import { Bsc } from '@bitgo/sdk-coin-bsc';
+import { Bnb } from '@bitgo/sdk-coin-bsc';
 
 const sdk = new BitGoAPI();
 
-sdk.register('bsc', Bsc.createInstance);
+sdk.register('bsc', Bnb.createInstance);
```
Comment (mmcshinsky-bitgo) [inline modules/sdk-coin-bsc/README.md:23]:
```suggestion
sdk.register('bnb', Bnb.createInstance);
```

**Comment by islamaminBitGo** [comment]
We want to be wary of a possible HSM change as well.

**Comment by mmcshinsky-bitgo** [comment]
Has there been consideration in not touching this module until it is fully deprecated and spinning up a new bnb module?

**Review by sachushaji — COMMENTED** [review]
Think we can keep the chain name as Binance smart chain itself, Not BNB smart chain, Other than that things looks good, Adding do not merge for this untill we made associated changes in all other Microservices. Think we have a lot of places to update inorder to have this updated everywhere and also we can consider @mmcshinsky-bitgo suggestion so that this becomes non-breaking

**Review by zahin-mohammad — COMMENTED** [review]
Flushing from review Q, please re-request review once this is ready.

File: modules/statics/src/coins.ts [hunk 3]
```diff
@@ -945,22 +945,22 @@ export const coins = CoinMap.fromCoins([
   ),
   account(
     'd0d44124-c7e9-4214-96ae-fbc6856ee3c2',
-    'bsc',
-    'Binance Smart Chain',
-    Networks.main.bsc,
+    'bnb',
+    'BNB Smart Chain',
```
[the remark this answers is not in the document]
Comment (kxl4126) [reply modules/statics/src/coins.ts:949]: Oh I think BSC is trying to distance itself from Binance exchange so they want to brand BSC (the chain) as BNB Smart Chain. That's what I read at least.

[the remark this answers is not in the document]
Comment (kxl4126) [reply modules/statics/src/coins.ts:949]: https://cointelegraph.com/learn/a-beginners-guide-to-the-bnb-chain-the-evolution-of-the-binance-smart-chain#:~:text=BNB%20vs.&text=The%20Binance%20Chain%2C%20where%20staking,Chain%2C%20still%20abbreviated%20as%20BSC.

The Binance Chain, where staking and voting occur (BNB Chain Governance), has changed its identity to BNB Beacon Chain under the rebranded BNB Chain umbrella. The Binance Smart Chain, which supports several blockchains and is compatible with the EVM, is known simply as BNB Smart Chain, still abbreviated as BSC.

**Comment by kxl4126** [comment]
> Has there been consideration in not touching this module until it is fully deprecated and spinning up a new bnb module?

I think this is good route to take --- I'll follow up with team.

**Pull request closed without merging**